src/SeleniumLibrary/__init__.py

```diff
@@ -42,133 +42,190 @@
+    | link         | Exact text a link has.              | ``link:The example``           |
+    | partial link | Partial link text.                  | ``partial link:he ex``         |
+    | sizzle       | Sizzle selector provided by jQuery. | ``sizzle:div.example``         |
+    | jquery       | Same as the above.                  | ``jquery:div.example``         |
+    | default      | Keyword specific default behavior.  | ``default:example``            |
+
+    See the `Default locator strategy` section below for more information
+    about how the default strategy works. Using the explicit ``default``
+    prefix is only necessary if the locator value itself accidentally
+    matches some of the explicit strategies.
+
+    Spaces around the separator are ignored, so ``id : foo``, ``id: foo``
+    and ``id:foo`` are all equivalent.
+
+    Examples:
+
+    | `Click Element` | id:container                      |
+    | `Click Element` | css:div#container h1              |
+    | `Click Element` | xpath: //div[@id="container"]//h1 |
+
+    Notice that using the ``sizzle`` strategy or its alias ``jquery``
+    requires that the system under test contains the jQuery library.
+
+    === Implicit XPath strategy ===
+
+    If the locator starts with ``//`` or ``(//``, the locator is considered
+    to be an XPath expression. In other words, using ``//div`` is equivalent
+    to using explicit ``xpath://div``.
+
+    Examples:
+
+    | `Click Element` | //div[@id="container"] |
+    | `Click Element` | (//div)[2]             |
+
+    The support for the ``(//`` prefix is new in SeleniumLibrary 3.0.
+
+    === Default locator strategy ===
+
+    By default locators are considered to use the keyword specific default
+    locator strategy. All keywords support finding elements based on ``id``
+    and ``name`` attributes, but some keywords support additional _key
+    attributes_ that make sense in their context. For example, `Click Link`
+    supports the link text and the ``href`` attribute in addition to the
+    normal ``id`` and ``name``.
+
+    Examples:
+
+    | `Click Element` | example | # Match based on ``id`` or ``name``.            |
+    | `Click Link`    | example | # Match also based on link text and ``href``.   |
+    | `Click Button`  | example | # Match based on ``id``, ``name`` or ``value``. |
+
+    If the locator accidentally starts with some of the explicit locator
+    prefixes or with the implicit XPath prefixes, it is possible to use
+    the explicit ``default`` prefix to enable the default strategy.
+
+    Examples:
+
+    | `Click Element` | name:foo         | # Find element with class ``foo``.               |
+    | `Click Element` | default:name:foo | # Use default strategy with value ``class:foo``. |
+    | `Click Element` | //foo            | # Find element using XPath ``//foo``.            |
+    | `Click Element` | //foo            | # Use default strategy with value ``//foo``.     |
```

Comment:
The locator is same in the above xpath example but the comment says something else, should the locator be just `foo` without the double forward slashes.

Reply:
It should be `default://foo`. Will fix.